.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -0,0 +1,43 @@
+name: "\U0001F41B Bug Report"
+description: Report an issue or possible bug
+title: "\U0001F41B BUG:"
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ##  Quick Checklist
+        Thank you for taking the time to file a bug report! Please fill out this form as completely as possible.
+
+        ✅ I am using the **latest version of Wrangler**.
+  - type: input
+    attributes:
+      label: What version of `Wrangler` are you using?
+      placeholder: 0.0.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: What package manager are you using?
+      placeholder: npm, yarn, pnpm
+    validations:
+      required: true
```

Comment:
Is this relevant to people facing problems with wrangler usage?

Reply:
Not necessarily, I was trying to figure out anything I thought could prove helpful but wasn't sure. 

I think overall simplifying the "guided" inputs as you mention in your bullet point suggestions is ideal. 

We can also have a big "comments" input field that users can use for any other details or messages they want to convey.